.changeset/many-moles-sing.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/backend-common': patch
+---
+
+Deprecate the legacy `createServiceBuilder`.
```

Comment:
Maybe a little bit more info here on what to do instead.

Reply:
Updated 🙌🏻